eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -0,0 +1,105 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $PullRequestNumber,
+  [Parameter(Mandatory=$True)]
+  [string] $BuildId,
+  [Parameter(Mandatory=$True)]
+  [string] $CommitSha,
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactList,
+  [string] $RepoFullName = "",
+  [string] $ArtifactName = "packages",
+  [string] $APIViewUri = "https://apiview.dev/PullRequest/DetectApiChanges"
+)
+
+# Submit API review request and return status whether current revision is approved or pending or failed to create review
+function Submit-Request($filePath)
+{
+    $repoName = $RepoFullName
+    if (!$repoName) {
+        $repoName = "azure/azure-sdk-for-$Language"
```

Comment:
```suggestion
        $repoName = "azure/azure-sdk-for-$LanguageShort "
```

I think you want LanguageShort here based on looking at https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/Language-Settings.ps1#L2

Reply:
Done